src/pykeen/pipeline/api.py

```diff
@@ -908,6 +919,15 @@ def pipeline(  # noqa: C901
         optimizer_kwargs,
         params=model_instance.get_grad_params(),
     )
+
+    lr_scheduler_instance = lr_scheduler_resolver.make(
```

Comment:
shouldn't we check `if lr_scheduler is not None:` before doing this, otherwise set `lr_scheduler_instance` to None?

Reply:
Good point. I'll add it